common-npm-packages/package.json

```diff
@@ -4,7 +4,8 @@
   "description": "This project welcomes contributions and suggestions.  Most contributions require you to agree to a\r Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us\r the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.",
   "main": "index.js",
   "scripts": {
-    "build": "cd ./build-scripts && npm install && cd .. && node make.js"
+    "build": "cd ./build-scripts && npm install && cd .. && node make.js",
+    "publish": "node publish.js"
```

Comment:
Is this file missing?

Reply:
The .gitignore got me! Should be there now